src/js/popup.js

```diff
@@ -0,0 +1,31 @@
+/* global document, window */
+const setUpAbout = () => {
+  const aboutElement = document.querySelector(".about-popup");
+  const infoButton = document.querySelector(".info-icon");
+
+  infoButton.addEventListener("click", () => {
+    aboutElement.style.display =
+      aboutElement.style.display !== "block" ? "block" : "none";
+  });
+
+  // closes window on clicks outside the info popup
+  window.addEventListener("click", (event) => {
+    if (
+      !infoButton.contains(event.target) &&
+      aboutElement.style.display === "block" &&
+      !aboutElement.contains(event.target)
+    ) {
+      aboutElement.style.display = "none";
+      infoButton.classList.toggle("active");
+    }
+  });
```

Comment:
I love this! I was using the Parking Lot map today on mobile and realized this is a bad UX currently that you have to scroll up to the small icon to close out of it.

Could you please add this improvement to PLM? Also feel free to reorganize the files so that `about.js` and `about.scss` are dedicated files. I like how you organize this.